src/core/storage/storagekeys.js

```diff
@@ -32,7 +32,7 @@ const StorageKeys = {
   SESSIONS_OPT_IN: 'sessions-opt-in',
   VERTICAL_PAGES_CONFIG: 'vertical-pages-config',
   LOCALE: 'locale',
-  SORT_BYS: 'sort-bys',
+  SORT_BYS: 'sortBys',
```

Comment:
for my own edification, what is the reason for this change?

Reply:
In the spec we wanted to have url params that mirror liveApis for setting stuff like sorts/filters on load. The api param is `sortBys` not `sort-bys`, so it seemed like the easiest way to do this was to do setWithPersists on `SORT_BYS`, instead of having a separate `PERSISTED_SORT_BYS` or something. The other PRs had to have new storage keys, because some transformations take place between the data in storage and the data sent to the api, but for `SORT_BYS` what we store in storage is exactly what is sent to the api, so it seemed like we could get away with this instead?

Reply:
ah, okay. that makes sense